spec/controllers/v0/veteran_readiness_employment_claims_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
     context 'visitor' do
       it 'validates successfully' do
         VCR.use_cassette 'veteran_readiness_employment/send_to_vre' do
-          form_params = { veteran_readiness_employment_claim: { form: test_form_no_vet_info.form } }
+          form_params = { veteran_readiness_employment_claim: { form: test_form.form } }
```

Comment:
why did this test change?

Reply:
Good question, after we ran the e2e tests and added more code to this PR, this test threw the good ol' undefined method for nil error. 

Turns out that we were allowing `nil` values through for the`veteranInformation` hash attribute in the form. The form we were passing here did not have `veteranInformation`.

Your question though made me think we should add validation for that so I added it in the model too.